krux_boto/boto.py

```diff
@@ -81,6 +90,38 @@ def get_boto(args=None, logger=None, stats=None):
         stats=stats,
     )
 
+def get_boto3(args=None, logger=None, stats=None):
+    """
+    Return a usable Boto3 object without creating a class around it.
+
+    In the context of a krux.cli (or similar) interface the 'args', 'logger'
+    and 'stats' objects should already be present. If you don't have them,
+    however, we'll attempt to provide usable ones for the boto setup.
+
+    (If you omit the add_boto_cli_arguments() call during other cli setup,
+    the Boto object will still work, but its cli options won't show up in
+    --help output)
+    """
+
+    if not args:
+        parser = get_parser()
+        add_boto_cli_arguments(parser)
+        args = parser.parse_args()
+
+    if not logger:
+        logger = get_logger(name=NAME)
+
+    if not stats:
+        stats = get_stats(prefix="boto")
```

Comment:
@hanpeter ok, happy to change as long as you're sure it's not affecting stats in production